### PR TITLE
Add figma contribution guide

### PR DESCRIPTION
**Pull request by lukasoppermann**
I added the figma contribution guidelines to the docs. 

[👀 Preview](https://primer-32c707d434-26441320.drafts.github.io/guides/figma/contribute)

Is there anything missing?

### CODE REVIEW

**Comment by emilybrick** [comment]
Hey @lukasoppermann - maybe we can catch up tomorrow! We [updated](https://github.com/primer/design/pull/352) the primer.style/design nav to match the new IA (most importantly, to allow for 2 levels of nesting in the nav). 

Posting links below for posterity/reference:
[Primer IA (ADR)](https://github.com/github/primer/blob/2023-01-10-primer-docs-IA/adrs/2023-01-10-primer-docs-IA.md)
[Primer IA spreadsheet](https://docs.google.com/spreadsheets/d/1MzdERnNsCeh_DAd5svKFBG9ojK9FDX4lgTkjCZjW2pI/edit#gid=0)
[Primer IA FigJam](https://www.figma.com/file/811c7oT909JpbnlLe7biXn/Primer-docs-IA?node-id=0%3A1&t=fjAasShNy8MZ4Ax2-1)

Would you be okay moving this Figma section to /guides? 

It would look something like this in the UI
Screen Shot 2023-01-19 at 1 40 02 PM


And this in nav.yml:
```
- title: Guides
  children:
    - title: Introduction
      url: /guides/introduction
    - title: Content
      url: /guides/content
    - title: The Zen of GitHub
      url: /guides/zen
    - title: Figma
      children:
        - title: Introduction
          url: /guides/figma/introduction
        - title: How to contribute
          url: /guides/figma/how-to-contribute
```

Alternatively, if you are planning to move the Figma component documentation in this section (until we have 'unified' component docs) - we can talk about keeping this section here temporarily.

**Comment by lukasoppermann** [comment]
Yes, this is fine. However atm, this nav is not live, right? So maybe we can merge this here and once the new nav gets merged you move it?

I'd like to get this info live asap, so people can find it.

**Comment by josepmartins** [comment]
Hey @lukasoppermann I think you have to update to the latest doctocat version 4.3.0 for the third level to be visible. Fixing the branch conflicts and updating the branch should work

**Comment by lukasoppermann** [comment]
> Hey @lukasoppermann I think you have to update to the latest doctocat version 4.3.0 for the third level to be visible. Fixing the branch conflicts and updating the branch should work

Thanks, I didn't run `npm i` before. Not it works fine. 👍

File: content/guides/figma/contribute.mdx
Comment (mperrotti) [inline content/guides/figma/contribute.mdx:10]: We don't usually keep assets like this in the repo. We just generate a URL by adding them to an issue comment.

File: src/@primer/gatsby-theme-doctocat/components/ImageBox.js
Comment (mperrotti) [inline src/@primer/gatsby-theme-doctocat/components/ImageBox.js:7]: Non-blocking nitpick: this could semantically be a `<figure>`

File: src/@primer/gatsby-theme-doctocat/components/ImageBox.js
Comment (mperrotti) [inline src/@primer/gatsby-theme-doctocat/components/ImageBox.js:26]: Non-blocking nitpick: this could semantically be a `<figcaption>`

File: src/@primer/gatsby-theme-doctocat/components/ImageBox.js
Comment (mperrotti) [inline src/@primer/gatsby-theme-doctocat/components/ImageBox.js:5]: Non-blocking nitpick: I personally prefer to visually associate the caption with the image using whitespace instead of borders and matching background colors. It feels a little quieter and cleaner to me, but I realize that's just personal preference.

Screenshot 2023-01-23 at 10 47 20 AM

**Review by mperrotti — APPROVED** [review]
Looking good! Just a few minor things to consider

File: content/guides/figma/contribute.mdx
Comment (lukasoppermann) [reply content/guides/figma/contribute.mdx:10]: Why is this better? I know we often do this, but it seems somehow "wrong" to me.

File: src/@primer/gatsby-theme-doctocat/components/ImageBox.js
Comment (lukasoppermann) [reply src/@primer/gatsby-theme-doctocat/components/ImageBox.js:5]: This is how it was in figma, but I am with you as long as we keep the bg behind the img. They feel lost without any bg somehow. Will update 👍

Comment (lukasoppermann) [reply src/@primer/gatsby-theme-doctocat/components/ImageBox.js:5]: Preview: https://primer-32c707d434-26441320.drafts.github.io/guides/figma/contribute

File: content/guides/figma/contribute.mdx
Comment (emilybrick) [reply content/guides/figma/contribute.mdx:10]: I think its just a lightweight way of storing images, but I don't know if I have the right answer. I always assumed it was because of perf and maintenance (as in, less to maintain in the repo?).

**Review by emilybrick — APPROVED** [review]

File: content/guides/figma/contribute.mdx
Comment (mperrotti) [reply content/guides/figma/contribute.mdx:10]: I personally _don't_ think it's better and would prefer keeping them in version control like you're doing here. It's just how we've been handling assets in our docs.

Maybe somebody who's been around longer knows the answer. @colebemis ?

Comment (emilybrick) [reply content/guides/figma/contribute.mdx:10]: As long as we are using static images, I think it's more important to make sure we are cataloging them in Figma in a way that people can easily locate and find the editable versions. That said - I don't have strong feelings about storing them in the repo.

Comment (lukasoppermann) [reply content/guides/figma/contribute.mdx:10]: Great I will merge and if @colebemis or @jonrohan have feedback we can still optimize the image later.

**Pull request merged**
